backend_typescript/models/GameManager.ts

```diff
@@ -0,0 +1,109 @@
+import { Result, Success, Failure } from "./Result";
+import * as db from '../db';
+import Game from "./Game";
+
+
+interface InvitedGames {
+    game_id: number;
+    opponent: string;
+    board_dimension: number;
+    createdByMe: boolean;
+}
+
+
+export interface FinalResults {
```

Comment:
Please call this interface `UserGameList` or similar. 
`FinalResults` is not a meaningful name.